test/boost/mutation_reader_test.cc

```diff
@@ -3883,8 +3883,7 @@ static future<> do_test_clustering_order_merger_sstable_set(bool reversed) {
         sstables::generation_type::int_t gen = 0;
         for (auto& mb: scenario.readers_data) {
             auto sst_factory = [table_schema, &env, &tmp, gen = ++gen] () {
-                return env.make_sstable(std::move(table_schema), tmp.path().string(), gen,
-                    sstables::sstable::version_types::md, sstables::sstable::format_types::big);
+                return env.make_sstable(std::move(table_schema), tmp.path().string(), gen);
```

Comment:
was: sstables::sstable::version_types::md
now: (defaulted to) sstables::get_highest_sstable_version() -> me

Reply:
yes, probably should have separated that.
the was no need to use `md` sepcifically.

Reply:
patch extracted in next version